genesis/engine/solvers/fem_solver.py

```diff
@@ -853,6 +886,7 @@ def linesearch(self, f: ti.i32):
 
     def batch_solve(self, f: ti.i32):
         self.batch_active.fill(True)
+        print("Batch solve for frame", f)
```

Comment:
No prints except in examples (eventually). You may use `gs.logger.debug` if you think it is worth it but I'm not convinced in this case.